docs/packages/pkg/libusb.rst

```diff
@@ -0,0 +1,29 @@
+.. spelling::
+
+    libusb
+
+.. index::
+  single: unsorted ; libusb
+
+.. _pkg.libusb:
+
+libusb
+======
+
+-  `Official <https://libusb.info>`__
+-  `Hunterized <https://github.com/libusb/libusb>`__
```

Comment:
It's not "Hunterized", it's "Official GitHub".

Reply:
Fixed.